app/views/users/import.haml

```diff
@@ -0,0 +1,7 @@
+%h1 Import User Data
+
+= form_with url: '#', local: true, multipart: true, authenticity_token: true do |form|
```

Comment:
Yeah, it's https://github.com/github/linguist, specifically the submodule for Ruby https://github.com/aroben/ruby.tmbundle. The actual Ruby code is fine.